fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -3,7 +3,7 @@
   enable_ruby
   <record>
     hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
-    message ${log}
+    message ${(message rescue nil) || log}
     version 1.1.4
   </record>
   remove_keys log,stream
```

Comment:
Under what conditions will a message have both `message` and `log`?  Can you provide an example?
Perhaps compare `message` and `log` - if they are the same, just keep `message`, otherwise keep both.  Also note that this may add to the amount of data per log entry in ES, if it is a frequent occurrence that an entry has both `log` and `message` and they are different.